### PR TITLE
Added support to pass multiple string params

### PR DESCRIPTION
**Pull request by rrgarciach**
Added support to pass multiple string params without breaking legacy implementations.

### CODE REVIEW

**Comment by rrgarciach** [comment]
Travis CI seems to be failing due an issue with rake ¿?

File: test/xsltproc_test.js [hunk 1]
```diff
@@ -10,10 +10,16 @@ it("should return the correct TXT", function (done) {
       xml = '../xsltproc/test/fixtures/data.xml',
       opts = {
           "output": "test/fixtures/test.html",
-          "stringparam": {
-             "key": 'title',
-             "val": 'This is a single parameter passed as subtitle----anvidsahviulasdhvklasdbcuw'
-          }
+          "stringparam": [
```
Comment (ilyar) [inline test/xsltproc_test.js]: need add this in another case (it)

**Review by ilyar — CHANGES_REQUESTED** [review]

**Comment by ilyar** [comment]
> Travis CI seems to be failing due an issue with rake ¿?

I'll check it later

File: test/xsltproc_test.js (hunk 1 above)
Comment (rrgarciach) [reply test/xsltproc_test.js]: You are absolutely right. 
I have set that test case as it was before and added the one for multiple string params.

**Comment by rrgarciach** [comment]
I've added additional test for multiple string parameters and also included a separated example with that case.

**Comment by ilyar** [comment]
@rrgarciach i'am fix ci https://github.com/ilyar/xsltproc/pull/4 please rebase your branch and squash duplicate commit

**Comment by rrgarciach** [comment]
@ilyar I can't see what's failing on travis-cli. Can you?

**Comment by ilyar** [comment]
fix branch in https://github.com/ilyar/xsltproc/pull/7
I'll watch the code tomorrow

**Comment by rrgarciach** [comment]
So @ilyar, do you want me to pull changes from #7 (branch "Pr/3") to my master branch so I can have this #3 PR (from branch master) updated... 
or do you want me to checkout to Pr/3 branch, merge/bring my changes in there so we can then merge to your master from that PR #7?
Sorry for asking, but I prefer to have all clear before messing things up

**Comment by ilyar** [comment]
@rrgarciach I created new PR because don't have write permission your branch. Your branch not fats fast forward.

> please rebase your branch and squash duplicate commit

```
git fetch
git rebase -i upstream/master
# in opened editor squash duplicate commit
git push -f
```
I made it self in PR3 and fix faild test.
Please make the final review of the code #7 so that we can merge your solution. (#3 will be canceled).

**Comment by ilyar** [comment]
@rrgarciach another approach

```
git fetch
git reset --hard upstream/pr/3
git push -f
```
in this case will pour your #3 request, on the other #7 canceled

**Comment by ilyar** [comment]
@rrgarciach Are my explanations clear?

**Comment by rrgarciach** [comment]
Closing this PR in favor of #7

**Pull request closed without merging**